pdf_build_src/process_markdowns.py

```diff
@@ -141,6 +141,164 @@ def modify_changelog():
         file.writelines(data)
 
 
+"""Number of chars maximal in one line approximated from a line of the PDF"""
+NB_CHARS_LINE_PDF = 100
+
+def correct_table(table):
+    """Compute the max number of characters and dashes for each column and create the corrected table
```

Comment:
the first line of a doc string should be a single sentence in imperative mode, ending with a dot.

https://numpydoc.readthedocs.io/en/latest/format.html